src/components/Element/ArticleElement/Tip.jsx

```diff
@@ -1,7 +1,13 @@
 import clsx from 'clsx';
+import { useEffect, useState } from 'react';
 
 export function Tip({ word, position, link }) {
   const { top, left } = position;
+  const [wordList, setWordList] = useState([]);
+
+  useEffect(() => {
+    console.log(wordList);
```

Comment:
don't leave console logs in production code

Reply:
removed.